README.md

```
@@ -341,6 +341,25 @@ crap4rs completions elvish > ~/.config/elvish/lib/crap4rs.elv
+## Claude Code skill (`/cut-the-crap`)
+
+crap4rs ships a reference [Claude Code](https://claude.com/claude-code) skill that consumes `--format advice` and drives a cover-then-split remediation loop on every over-threshold function. It lives in [`skills/cut-the-crap/`](skills/cut-the-crap/) — copy it into your user skills directory once and it's available from any Claude Code session:
+
+```bash
+# from the repo root
+cp -r skills/cut-the-crap ~/.claude/skills/
+```
+
+Then in any Claude Code session:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)